muxers/mplex/src/config.rs

```diff
@@ -84,6 +88,17 @@ impl MplexConfig {
         self.split_send_size = size;
         self
     }
+
+    /// Gets protocol
+    pub fn protocol(&self) -> &[u8] {
+        self.protocol
+    }
+
+    /// Sets protocol, muxer.set_protocol(b"/mplex/6.7.0")
```

Comment:
```suggestion
    /// Set protocol name.
    ///
    /// ```rust
    /// muxer_config.set_protocol(b"/mplex/6.7.0");
    /// ```
```